src/main/java/bio/terra/cli/command/app/passthrough/Gcloud.java

```diff
@@ -1,11 +1,33 @@
 package bio.terra.cli.command.app.passthrough;
 
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.exception.PassthroughException;
+import bio.terra.cli.exception.UserActionableException;
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the second-level "terra gcloud" command. */
 @Command(name = "gcloud", description = "Call gcloud in the Terra workspace.")
 public class Gcloud extends ToolCommand {
 
+  private static final Logger logger = LoggerFactory.getLogger(Git.class);
+
+  @CommandLine.Option(
+      names = "--gcs-bucket",
+      description = "specify the gcs buket for gcloud builds")
+  public String bucketName;
+
+  @CommandLine.Option(names = "builds", description = "")
```

Comment:
25-29 are awkward. Could this be used instead?

https://cs.github.com/DataBiosphere/terra-cli/blob/dfcad196d73e5ffcaafc615e993361d943792020/src/main/java/bio/terra/cli/command/app/Execute.java?q=parameters#L24

You would make `command` public.